### PR TITLE
Added class files

### PR DESCRIPTION
**Pull request by Joanna-G**

### CODE REVIEW

File: Mailbox.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import java.util.ArrayList;
+
+public class Mailbox {
+	
+	private ArrayList<Message> messageQueue = new ArrayList();
```
Comment (t-walker-21) [inline Mailbox.java:5]: I think that Dr. Li wants us to use JavaDoc to document our code. Could you add more comments?

File: Message.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (t-walker-21) [inline Message.java:1]: Same as first comment, try to conform to JavaDoc standard comments.

File: Message.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+public class Message {
+	
+	private String toAddress;
+	private String fromAddress;
+	private String message;
+	
+	public void getTo(String toAddress) {
```
Comment (t-walker-21) [inline Message.java]: Traditionally, get/set methods are used to either 

- return a value contained in a private member

- set the value within a private member

It looks like you're trying to use a get method to set a member value, when _instead_ you should use a _set_ method to assign a value to private member.

Use get methods to return the value that is contained within the private member

**Comment by Joanna-G** [comment]
Eventually, yes. Code isn't really part of what is required for the first part, so this is just the very basics of what I had so far. I will add more later.

File: Message.java (hunk 3 above)
Comment (Joanna-G) [reply Message.java]: Eventually, yes. Code isn't really part of what is required for the first part, so this is just the very basics of what I had so far. I will add more later.

**Review by t-walker-21 — COMMENTED** [review]
I like that you've already started writing code and familiarized yourself with Git. Looks like you learned it way quicker than I did! What I want to do is create a workspace folder that will be in our repository, so that we can keep an entire project in GH as opposed to individual classes. I will create a workspace within GH and add your classes. Great work, Joanna!

File: Message.java (hunk 3 above)
Comment (t-walker-21) [reply Message.java]: Okay, were you just kind of outlining what your ideas/use cases were?

Comment (Joanna-G) [reply Message.java]: I always get the get/set confused. lol. Yeah, just kinda throwing some stuff together for now. Refine as we go, I figure.

Comment (t-walker-21) [reply Message.java]: http://stackoverflow.com/questions/6638964/set-and-get-methods-in-java

Data protection can get confusing sometimes. I'll create a project  and push to master tonight with your classes added after you fix some of those issues.

File: Message.java [hunk 4]
```diff
@@ -5,15 +5,15 @@
 	private String fromAddress;
 	private String message;
 	
-	public void getTo(String toAddress) {
+	public void setTo(String toAddress) {
```
Comment (t-walker-21) [inline Message.java]: Perfect

**Pull request closed without merging**